applications/nrf5340_audio/README.rst

```diff
@@ -615,6 +615,14 @@ Only one of the following :file:`.conf` files is included when building:
   When building using the command line, you must explicitly specify if :file:`prj_release.conf` is going to be included instead of :file:`prj.conf`.
   See :ref:`nrf53_audio_app_building` for details.
 
+Requirements for FOTA
```

Comment:
If this is the first time FOTA is mentioned, we should explain the abbreviation.

Reply:
Done